package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nw-style-guide",
-  "version": "3.1.5-beta.0",
+  "version": "3.2.0",
```

Comment:
This version looks ok to me but I don't know why you published 3.3.0-1? You might want to "unpublish" this version